app/handlers/objectTracking.py

```diff
@@ -17,5 +27,56 @@ class ObjectTrackingHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+
     def post(self):
+        self.objectTrack(self.request.body_arguments["identifier"])
+        
+        message = "Hello,\n\tWe have finished processing your video and identifying all objects.\nThank you for your patience,\nThe Santos Team"
+        subject = "Your video has finished processing."
+
+        EmailHelper.send_email(self.request.body_arguments["email"], subject, message)
+
         self.finish("Object Tracking")
+
+
+    def objectTrack(self, identifier):
+        """
+        Runs TrafficIntelligence trackers and support scripts.
+        """
+        project_path = get_project_path(identifier)
+
+        # create test folder
+        if not os.path.exists(os.path.join(project_path, "run")):
+            os.mkdir(os.path.join(project_path, "run"))
+
+        tracking_path = os.path.join(project_path, "run", "run_tracking.cfg")
+
+        # removes object tracking.cfg
+        if os.path.exists(tracking_path):
+            os.remove(tracking_path)
+
+        # creates new config file
+        prev_tracking_path = os.path.join(project_path, ".temp", "test", "test_object", "object_tracking.cfg")
+        shutil.copyfile(prev_tracking_path, tracking_path)
```

Comment:
Not 100% sure if we're doing good things with files here. Why do we check for the /run folder, and then assume that /run/run_tracking exists? Why do we delete the old object tracking, and replace it?

(IMO, the change that should be make here is deleting the check for if /run exists. If it doesn't exist, that check doesn't help anyway because run_tracking won't exist).